### ISSUE TITLE
Add automake to the full stack for gems that have C code dependencies.

### ISSUE THREAD

**Issue by genevieve** [paketo-buildpacks/stacks#29]
Hi! I have a ruby app that I'm trying to build with:
```
pack build app --builder paketobuildpacks/builder:full --buildpack gcr.io/paketo-buildpacks/ruby
```

And it fails with:
```
Paketo Bundle Install Buildpack 0.0.59
  Executing build process
    Running 'bundle config path /layers/paketo-buildpacks_bundle-install/gems'
    Running 'bundle config cache_path --parseable'
    Running 'bundle install --local'
failed to execute bundle install output:
Resolving dependencies...
Installing rake 12.3.3
...
Installing snappy 0.0.17 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
current directory:
/layers/paketo-buildpacks_bundle-install/gems/gems/snappy-0.0.17/ext
/layers/paketo-buildpacks_mri/mri/bin/ruby -I
/layers/paketo-buildpacks_mri/mri/lib/ruby/2.6.0 -r
./siteconf20201016-71-o8igwx.rb extconf.rb
checking for -lsnappy... no
./autogen.sh: 3: ./autogen.sh: aclocal: not found
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.
Provided configuration options:
	--with-opt-dir
	--without-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/layers/paketo-buildpacks_mri/mri/bin/$(RUBY_BASE_NAME)
	--with-libsnappy-config
	--without-libsnappy-config
	--with-pkg-config
	--without-pkg-config
	--with-snappylib
	--without-snappylib
extconf.rb:10:in `block in <main>': `autogen.sh` failed (RuntimeError)
	from extconf.rb:9:in `chdir'
	from extconf.rb:9:in `<main>'
To see why this extension failed to compile, please check the mkmf.log which can
be found here:
/layers/paketo-buildpacks_bundle-install/gems/extensions/x86_64-linux/2.6.0-static/snappy-0.0.17/mkmf.log
extconf failed, exit code 1
Gem files will remain installed in
/layers/paketo-buildpacks_bundle-install/gems/gems/snappy-0.0.17 for inspection.
Results logged to
/layers/paketo-buildpacks_bundle-install/gems/extensions/x86_64-linux/2.6.0-static/snappy-0.0.17/gem_make.out
An error occurred while installing snappy (0.0.17), and Bundler cannot continue.
Make sure that `gem install snappy -v '0.0.17' --source XXX` succeeds before bundling.
In Gemfile:
.. which depends on
      snappy
error: exit status 5
ERROR: failed to build: exit status 1
ERROR: failed to build: executing lifecycle: failed with status code: 145
```

I believe the gem is trying to compile some c code but the stack doesn't have automake so it fails. Would it be possible to add automake?

**Comment by mdelillo** [comment]
Hey @genevieve, it looks like this issue can be resolved by using version `0.1.0` or later of snappy. Does that work for you?

**Comment by mdelillo** [comment]
PR merged, the new full stack will be published shortly
